### ISSUE TITLE
Explore exposing a proper "terminal" (like what telnet does, instead of netcat)?

### ISSUE THREAD

**Issue by iagox86** [iagox86/dnscat2#76]
It'd be really cool if the "shell" sessions would be able to be full-on terminals, with colours and the ability to run vim and stuff like that.

I don't really know what ssh does that's different than what netcat does, but it'd be good to explore it and see if I can make it happen!

**Comment by bmenrigh** [comment]
Yes exposing a TTY would be amazing.  I've tried variations on exec --command='/usr/bin/python2.7 -c import pty; pty.spawn("/bin/bash");' without success.
